src/Observable.ts

```diff
@@ -247,7 +247,7 @@ export class Observable<T> implements CoreOperators<T>  {
   publishBehavior: (value: any) => ConnectableObservable<T>;
   publishReplay: (bufferSize?: number, windowTime?: number, scheduler?: Scheduler) => ConnectableObservable<T>;
   publishLast: () => ConnectableObservable<T>;
-  race: (...observables: Array<Observable<any>>) => Observable<any>;
+  race: (...observables: Array<Observable<T>>) => Observable<T>;
```

Comment:
@david-driscoll 's correct. Further change might be required to full support of implementation perspective, shortly I think for now this signature's more correct to avoid any possible confusion from consumer point.